### PR TITLE
Update Kevin Malone's ReadMe

### PR DESCRIPTION
**Pull request by amrro**
## Linear Ticket

- [ENG-927 - Improve docs for Kevin Malone set up](https://linear.app/govrn/issue/ENG-927/improve-docs-for-kevin-malone-set-up)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,13 +19,14 @@ Our monorepo uses [Nx](https://nx.dev/getting-started/intro) as a build system a
 
 ## Monorepo Structure
 
-Each app or lib has its own readme that explains the scoped purpose & required `.env` variables to operates. You have to create a `.env` file in each module root and provide the required variables mentioned in its README.
+Each app or lib has its own readme that explains the scoped purpose & required `.env` variables to operate.
```
Comment (alexkeating) [inline README.md:22]: require instead  of required

**Review by alexkeating — COMMENTED** [review]
Looks good one small change

File: apps/kevin-malone/README.md [hunk 2]
```diff
@@ -12,27 +12,40 @@ A self reporting bot for the movement model
...
 1. Clone this repository
-1. Create a virtual environment (`python3.9 -m venv ./.venv`)
-1. Activate the virtual environment (`source ./.venv/bin/activate`)
-1. Install dependencies via pip (`pip install -r requirements-dev.txt`)
-1. Setup environment variables as required in ./.env (see env.sample) for the values which need populating
-1. In a separate directory, clone the [Govrn monorepo](https://github.com/Govrn-HQ/govrn-monorepo), and follow the the setup instructions there. 
+2. Create a [virtual environment](https://python-poetry.org/docs/managing-environments)
+   with python `3.8`.
+3. Install dependencies via pip: `poetry run pip install -r requirements-dev.txt`.
+4. Setup environment variables: `cp env.example .env`
```
Comment (Flip-Liquid) [inline apps/kevin-malone/README.md:18]: Can we change this section to "Dev setup"? and just explain what is needed in order to build kevin, and run the tests? We can explain the requirements about postgres, protocol-api server, redis, and api keys/env in a separate "run locally" section. We'd just have to move steps 4 + 5

File: apps/kevin-malone/README.md [hunk 3]
```diff
@@ -12,27 +12,40 @@ A self reporting bot for the movement model
...
+5. Run protocol API server. Follow these [instructions](../protocol-api/README.md) for instructions (_update .env with
+   protocol API URL_).
 
+### Setup local redis instance
```
Comment (Flip-Liquid) [inline apps/kevin-malone/README.md:22]: Let's move steps 4 + 5 into their own ### sections under a new ## section "Setup Kevin Malone Locally"

File: apps/kevin-malone/README.md (hunk 2 above)
Comment (amrro) [reply apps/kevin-malone/README.md]: Requirements of protocol api is explained in protocol api's readme.
I've expanded the requirements section and moved the last couple of steps to "Run Locally" Section.

**Pull request merged**